automation/MPLogger.py

```diff
@@ -265,10 +263,22 @@ def _handle_serialized_writes(self, obj):
         This is currently records that are written to a file on disk
         and those sent to Sentry.
         """
+        def ensure_str(s):
+            """Ensures we get a string
+
+                Stolen and simplified from six
```

Comment:
This looks right, but please just clean up this comment to "Taken and simplified from the six library".

Reply:
In fact I think this is simple enough that we don't need to reference six.